components/brave_rewards/resources/extension/brave_rewards/utils.ts

```diff
@@ -23,6 +25,24 @@ export const formatConverted = (converted: string, currency: string = 'USD'): st
   return `${converted} ${currency}`
 }
 
+export const handleContributionAmount = (amount: string) => {
+  let result = '0.0'
+  const amountSplit = amount.split('.')
+  if (amountSplit && amountSplit[0].length > 18) {
+    const result = new BigNumber(amount).dividedBy('1e18').toFixed(1, BigNumber.ROUND_DOWN)
```

Comment:
Monetary amounts are normally rounded up, is this correct to round down?

Reply:
that is correct

Reply:
Based upon being correct, see comment below for test

Reply:
code was copied from an old code that handled notifications before https://github.com/brave/brave-core/pull/4237/files#diff-79335bdd00f8d2118de50c4a4cc18a13L28

Reply:
done